stat/stat_test.go

```diff
@@ -63,6 +63,42 @@ func TestCircularMean(t *testing.T) {
 	}
 }
 
+func ExampleCircularStdDev() {
+	x := []float64{0, 0.25 * math.Pi, 0.75 * math.Pi}
+	cstd := CircularStdDev(x)
+
+	fmt.Printf("The circular standard deviation is %.5f.\n", cstd)
+	// Output:
+	// The circular standard deviation is 1.04815.
+}
+
+func TestCircularStdDev(t *testing.T) {
+	for i, test := range []struct {
+		x   []float64
+		ans float64
+	}{
+		{
+			x:   []float64{0, 2 * math.Pi},
+			ans: 0,
+		},
+		// This test case taken from https://stackoverflow.com/questions/55616697/how-to-calculate-standard-deviation-of-circular-data :
+		{
+			x:   []float64{6.108655, 0.3490660, 0.698132},
+			ans: 0.36157912052347674,
+		},
+		// This test case taken from scipy's test suite:
```

Comment:
`s/:$/./`

Reply:
Done in 64989b26fd23c90ac9b46acf6cce313a236a6915.